src/coreclr/jit/importer.cpp

```diff
@@ -11204,6 +11210,83 @@ bool Compiler::impReturnInstruction(int prefixFlags, OPCODE& opcode)
     return true;
 }
 
+#ifdef DEBUG
+//------------------------------------------------------------------------
+// impPoisonImplicitByrefsBeforeReturn:
+//   Spill the stack and insert IR that poisons all implicit byrefs.
+//
+void Compiler::impPoisonImplicitByrefsBeforeReturn()
```

Comment:
Nit: a little comment for why do we want to do this?

Reply:
Done!